tox.ini

```diff
@@ -38,16 +38,16 @@ setenv =
     API_CLIENT_RUN_ON_REDIS = true
 allowlist_externals =
     git
-    /bin/cp
+    /bin/ln
     /bin/rm
 commands_pre =
     pip install -e .
     git clone {env:INTEGRATION_TESTS_REPO:git@github.com:gridsingularity/gsy-backend-integration-tests.git} {envtmpdir}/gsy-backend-integration-tests
-    cp -r {envtmpdir}/gsy-backend-integration-tests/gsy_e_sdk_tests/integration_tests/ ./integration_tests/
+    ln -sf {envtmpdir}/gsy-backend-integration-tests/gsy_e_sdk_tests/integration_tests/ ./integration_tests
```

Comment:
I do not understand why softlinks are better here. But if it works, LGTM

Reply:
Please explain the reason here, thanks! (could not find the conversation about it)

Reply:
That was my suggestion, and it was admittedly minor. It is in order to to avoid the time that it will take to copy the files In this case the time is negligible, however it is better as a practice to not copy files if one can avoid it and use symbolic links instead.

Reply:
Ah ok, makes sense. Thanks!